test/oatpp/web/PipelineTest.cpp

```diff
@@ -176,7 +176,7 @@ void PipelineTest::onRun() {
     pipeOutThread.join();
     pipeInThread.join();
 
-  }, std::chrono::minutes(10));
+  }, std::chrono::minutes(1));
```

Comment:
Let's keep this as it previously was. It's not a part of this PR.

Reply:
This was just part of debugging the tests. The pipeline-tests always time out on my machine (Debian 11) and I decreased it out of curiosity. Has no relation to the PR and will be squashed away. (If the PR will be merged, I would recommend an squash-merge anyway).